artifacts/example/overridepolicy_image.yaml

```diff
@@ -7,12 +7,13 @@ spec:
   resourceSelectors:
     - apiVersion: apps/v1
       kind: Deployment
-  targetCluster:
-    labelSelector:
-      matchLabels:
-        location: us
-  overriders:
-    imageOverrider:
-      - component: Registry
-        operator: replace
-        value: fictional.registry.us
+  overrideRules:
+    - targetCluster:
+        labelSelector:
+          matchLabels:
+            location: us
+      overriders:
+        imageOverrider:
+          - component: Registry
+            operator: replace
+            value: fictional.registry.us
```

Comment:
```
apiVersion: policy.karmada.io/v1alpha1
kind: OverridePolicy
metadata:
  name: example-override
  namespace: default
spec:
  resourceSelectors:
  - apiVersion: apps/v1
    kind: Deployment
  overrideRules:
  - targetCluster:
      labelSelector:
        matchLabels:
          location: us
    overriders:
      imageOverrider:
      - component: Registry
        operator: replace
        value: fictional.registry.us
  - targetCluster:
      labelSelector:
        matchLabels:
          location: cn
    overriders:
      imageOverrider:
      - component: Registry
        operator: replace
        value: fictional.registry.cn
```
Show overridepolicy's  excellent features that you contribute. We can support multi cluster override in one overridepolicy resource.

Reply:
Nice proposal! I will update this example.